Verbatim `util/nondecreasing.v` section of patches/prosa-v06-rocq93-official-proof-closure-common.patch
(sha256 6819bfa9ac0bb9cfd6bfc7182d57e22463cf7a57b38ce9af89c8e7de96868050), split out for classic official closures.
--- a/util/nondecreasing.v
+++ b/util/nondecreasing.v
@@ -2,6 +2,7 @@
 Require Export prosa.util.epsilon.
 Require Export prosa.util.nat.
 Require Export prosa.util.list.
+Set SsrOldRewriteGoalsOrder.
 
 (** * Non-Decreasing Sequence and Distances *)
 (** In this file we introduce the notion of a non-decreasing sequence. *)
